ros/api/v0/hosts.py

```diff
@@ -72,6 +72,11 @@ def get(self):
         # Refer - https://www.postgresql.org/docs/13/queries-limit.html
 
         account_query = db.session.query(RhAccount.id).filter(RhAccount.account == ident['account_number']).subquery()
+        system_count = db.session.query(System.id)\
+            .filter(System.account_id.in_(account_query)).count()
+        if system_count <= 0:
+            abort(404)
```

Comment:
I guess this response status code might conflict with other implementations( for example - RBAC). 
I need to figure out if  any other insights service is having similar scenario and uses any particular status code? OR suitable response code to avoid conflict.

Reply:
I have removed `404` handling inside `/systems` API and introduced new API `is_configured`.